src/BlockTypes/Checkout.php

```diff
@@ -24,6 +24,31 @@ class Checkout extends AbstractBlock {
 	 */
 	protected $chunks_folder = 'checkout-blocks';
 
+	/**
+	 * Initialize this block type.
+	 *
+	 * - Hook into WP lifecycle.
+	 * - Register the block with WordPress.
+	 */
+	protected function initialize() {
+		parent::initialize();
+		add_action( 'wp_loaded', array( $this, 'register_patterns' ) );
+	}
+
+	/**
+	 * Register block pattern for Empty Cart Message to make it translatable.
+	 */
+	public function register_patterns() {
+		register_block_pattern(
+			'woocommerce/checkout-heading',
+			array(
+				'title'    => '',
+				'inserter' => false,
+				'content'  => '<!-- wp:heading {"align":"wide", "level":1} --><h1 class="wp-block-heading alignwide">' . esc_html__( 'Checkout', 'woo-gutenberg-products-block' ) . '</h1><!-- /wp:heading -->',
```

Comment:
According to https://developer.wordpress.org/block-editor/reference-guides/components/heading/, the default heading level is 2. Using `<!-- wp:heading {"align":"wide"} -->` instead of `<!-- wp:heading {"align":"wide", "level":1} -->` would lead to the following error in the console:

<img width="784" alt="Screenshot 2023-07-20 at 12 50 57" src="https://github.com/woocommerce/woocommerce-blocks/assets/3323310/87c1d9cb-4ecd-4fa0-ba8a-51d5e533d8a5">

Reply:
See https://github.com/woocommerce/woocommerce-blocks/pull/10281#discussion_r1269332569